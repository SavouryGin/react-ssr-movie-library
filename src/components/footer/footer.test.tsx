import Footer from './index';
import React from 'react';
import { render, screen } from '@testing-library/react';

describe('Footer component:', () => {
  it('renders the footer container', () => {
    render(<Footer />);
    const footer = screen.getByRole('contentinfo');
    expect(footer).toBeInTheDocument();
  });

  it('matches the snapshot', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
