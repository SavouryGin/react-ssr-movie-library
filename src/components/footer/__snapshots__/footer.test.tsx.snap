// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component: matches the snapshot 1`] = `
<DocumentFragment>
  <footer
    class=""
  >
    <span>
      <strong>
        netflix
      </strong>
      roulette
    </span>
  </footer>
</DocumentFragment>
`;
